browser/templatetags/app_filters.py

```diff
@@ -141,3 +141,16 @@ def get_affiliates(institution):
     #  absolute URL.
 
     return qs
+
+
+@register.filter
+def get_positions(person):
+    return Position.objects.filter(person=person).order_by('year')
+
+
+@register.simple_tag
+def get_roles_of_positions():
+    list_of_roles = []
+    for i in Position.role_choices:
+        list_of_roles.append(i[1])
+    return list_of_roles
```

Comment:
using list comprehension this should be a one-liner

Reply:
Reduced the above code to a single line